### ISSUE TITLE
Suspend program until #175 fixed

### ISSUE THREAD

**Issue by friedger** [blockstack/app-mining#177]
According to 9. of [app mining terms](https://app.co/mining/terms), this is possible.

The November payout probably needs to happen nevertheless if I understand the T&C correctly.

**Comment by talhasch** [comment]
Currently there are too few dimensions to evaluate an application good or not. It sounds sensible to pause app mining until community decide on new dimensions / criterias.

**Comment by friedger** [comment]
The question of this issue is whether it is **more urgent/important** 
* to continue the app mining program as is or
* to fix the issue about rewarding apps that do not respect the users' data.

I still believe we can fix #175 within the next 7 days (before new app mining period starts) but the current interaction suggests that it won't be resolved by then.

**Comment by joberding** [comment]
Agreed. This issue is creating a potential culture problem wherein non-privacy respecting apps are obtaining benefits through app mining while privacy respecting apps are not being recognized.

**Comment by friedger** [comment]
See also an investors view: https://www.reddit.com/r/blockstack/comments/e3dc9i/blockstack_is_a_sinking_ship_and_it_needs_better/
